components/org.wso2.carbon.identity.branding.preference.management.core/pom.xml

```diff
@@ -88,6 +88,44 @@
             <artifactId>jackson-core</artifactId>
             <scope>provided</scope>
         </dependency>
+
+        <!-- Testing related dependencies -->
+        <dependency>
+            <groupId>org.testng</groupId>
+            <artifactId>testng</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.jacoco</groupId>
+            <artifactId>org.jacoco.agent</artifactId>
+            <classifier>runtime</classifier>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.powermock</groupId>
+            <artifactId>powermock-module-testng</artifactId>
```

Comment:
Let's remove the power mock. This is not working properly with latest JDK versions and we are soon migrating to JDK 17

Reply:
Powermock features are also not used in the test cases as it seems. So it would be easy to update.

Reply:
Addressed with https://github.com/wso2-extensions/identity-branding-preference-management/pull/4/commits/51db25e0eff78eb9a6b795904e020c10992e25a8